src/cookbook/design/themes.md

```diff
@@ -53,89 +86,113 @@ MaterialApp(
 );
 ```
 
-See the [`ThemeData`][] documentation to see all of
-the colors and fonts you can define.
+Most instances of `ThemeData` set values for the following two properties. These properties affect the entire app.
 
-## Themes for part of an application
+1. [`colorScheme`][] defines the colors.
+1. [`textTheme`][] defines text styling.
 
-To override the app-wide theme in part of an application,
-wrap a section of the app in a `Theme` widget.
+[`colorScheme`]: {{site.api}}/flutter/material/ThemeData/colorScheme.html
+[`textTheme`]: {{site.api}}/flutter/material/ThemeData/textTheme.html
 
-There are two ways to approach this: creating a unique `ThemeData`,
-or extending the parent theme.
+To learn what colors and fonts you can define,
+check out the [`ThemeData`][] documentation.
 
-{{site.alert.note}}
-  To learn more, watch this short Widget of the Week video on the Theme widget:
+## Apply a theme
 
-  <iframe class="full-width" src="{{site.youtube-site}}/embed/oTvQDJOBXmM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-{{site.alert.end}}
+To apply your new theme, add the `Theme.of(context)` method within a widget's styling properties. These include, but are not limited to, `style` and `color`.
+
+The `Theme.of(context)` method looks up the widget tree and retrieves
+the nearest `Theme` in the tree.
+If you have a standalone `Theme` as defined in the previous example,
+that's applied.
+If not, Flutter applies the app's theme.
+
+In fact, the `Container` uses this technique to find its `color``.
```

Comment:
I think the original use "In fact" was referring to Flutter's implementation of `FloatingActionButton`, not the following code. `Container` itself doesn't use `Theme.of` as far as I know.

Perhaps you can adjust this to refer to the implementation of `FloatingActionButton` and add a new sentence introducing the next code snippet?